graphkb/constants.py

```diff
@@ -19,9 +19,10 @@
 # the order here is the order these are applied, the first category matched is returned
 RELEVANCE_BASE_TERMS: CategoryBaseTermMapping = [
     ('therapeutic', BASE_THERAPEUTIC_TERMS),
-    # ('pharmacogenomic', ['metabolism', 'toxicity', 'dosage']) TODO: add when new pharmacogenomic section ready,
     ('diagnostic', ['diagnostic indicator']),
     ('prognostic', ['prognostic indicator']),
+    ('pharmacogenomic', ['metabolism', 'toxicity', 'dosage']),
+    ('cancer predisposition', ['pathogenic']),
```

Comment:
Would it make more sense to use "predisposing" as the top level term here?

![image](https://user-images.githubusercontent.com/4342641/151239137-c385ccbf-3adc-4411-833e-a00f7e40a445.png)

Reply:
From the comments on GERO-272, it seems like 'pathogenic' is desired as separate from the predisposing variant.

Reply:
Note for releasing: 'cancer predisposition' category is a breaking change for client.  Only works on the dev version right now:
```
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://iprstaging-api.bcgsc.ca/api/reports There was an error validating the report content Error: .kbMatches[0].category should be equal to one of the allowed values [allowedValues: therapeutic,prognostic,diagnostic,biological,unknown,novel,pharmacogenomic]
```